amass/amass.go

```diff
@@ -178,6 +188,44 @@ loop:
 	return nil
 }
 
+// DNSQueriesPerSec returns the number of DNS queries the enumeration has performed per second.
+func (e *Enumeration) DNSQueriesPerSec() int {
+	e.metricsLock.Lock()
+	defer e.metricsLock.Unlock()
```

Comment:
You're only ready the value here, consider using a [RWMutex](https://godoc.org/sync#RWMutex).

Reply:
Will do. Thanks!